### PR TITLE
Initial list for Definition of Done 

### PR DESCRIPTION
**Pull request by iteles**
Starts off our definition of done. It will require additional detail as what is unclear emerges throughout the week.
This ensures that there is an initial draft in place for summer contracts to be signed.

### CODE REVIEW

File: structure-of-work.md [hunk 1]
```diff
@@ -12,5 +12,20 @@ You acknowledge and agree that:
 
 
 ## Definition of "_Done_"
```
Comment (nelsonic) [inline structure-of-work.md:14]: What does "**_structure of work**_" _mean_?

Comment (iteles) [inline structure-of-work.md:14]: This is referenced from the freelance agreements and my initial thought was to have some more information on structure our work and how we work together within the same file. 
Following this week it's pretty apparent the definition of done needs to be split out into its own file.

File: structure-of-work.md [hunk 2]
```diff
@@ -12,5 +12,20 @@ You acknowledge and agree that:
...
+We will keep adding detail to this as we gain understanding of any ambiguities that may arise.
 
-> TBD: https://github.com/dwyl/start-here/issues/33
\ No newline at end of file
+For clarity, a task will not be considered to be complete until all these items can be checked off.
+- [ ] Time to be taken for the task was estimated and agreed by all present
```
Comment (nelsonic) [inline structure-of-work.md:18]: Why so many words?

> "**_Estimate expected time**_ required to complete task" (_before starting the task_)

**Comment by nelsonic** [comment]
@iteles any chance you can migrate this to its own repo and simply put a link in _both_ Summer-2015 and https://github.com/dwyl/start-here ?

**Pull request merged**